client/my-sites/checkout/checkout/index.jsx

```diff
@@ -318,6 +318,7 @@ export class Checkout extends React.Component {
 			if ( isPrivate ) {
 				cartItem = null;
```

Comment:
Could we simplify this logic by adding a return here?

It feels tricky to reason through right now and I think if you did that else clause can be simplified or you could make a conditional that is and if ... else if ... else.

Reply:
good suggestion- I simplified the logic